core/src/providers/openai.rs

```diff
@@ -31,6 +31,7 @@ use std::sync::Arc;
 use std::time::Duration;
```

Comment:
Use [this link](https://gist.github.com/davidebbo/b21a4171d8ee864757eca82298788278/revisions?diff=unified&w) to compare from before my original changes.